.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dsdl"]
```

Comment:
I'm a little conflicted about this. Conceptually dsdl depends on a give version of the specification which makes this, technically, a circular dependency. I'm also not sure that we want to treat the standard, regulated type documentation so differently. i.e. we might want to provide your render-to-latex script as something any dsdl repo can use to generate documentation.